drivers/sensor/ti/tmag5273/tmag5273.c

```diff
@@ -144,23 +144,24 @@ static int tmag5273_check_device_status(const struct tmag5273_config *drv_cfg,
 	}
 
 	if ((*device_status & TMAG5273_VCC_UV_ER_MSK) == TMAG5273_VCC_UV_ERR) {
-		LOG_WRN("VCC undervoltage detected");
+		LOG_ERR("VCC under voltage detected");
 	}
 #ifdef CONFIG_CRC
 	if (drv_cfg->crc_enabled &&
 	    ((*device_status & TMAG5273_OTP_CRC_ER_MSK) == TMAG5273_OTP_CRC_ERR)) {
-		LOG_WRN("OTP CRC error detected");
+		LOG_ERR("OTP CRC error detected");
 	}
 #endif
 	if ((*device_status & TMAG5273_INT_ER_MSK) == TMAG5273_INT_ERR) {
-		LOG_WRN("INT pin error detected");
+		LOG_ERR("INT pin error detected");
 	}
 
 	if ((*device_status & TMAG5273_OSC_ER_MSK) == TMAG5273_OSC_ERR) {
-		LOG_WRN("Oscillator error detected");
+		LOG_ERR("Oscillator error detected");
 	}
 
-	return 0;
+	LOG_ERR("unknown error");
```

Comment:
We'd hit this error even when the error is known.
```suggestion
```

Reply:
good catch! removed it.